fix: branding for search page buttons

Enubia opened the pull request:
added cdn links back as comments so that we can revert the static
serving in case this doesn't work

gnorbsl commented:
this might clash with my branch where I updated the scripts and added the map files, can we merge mine first and see if there are conflicts?

Enubia commented:
> this might clash with my branch where I updated the scripts and added the map files, can we merge mine first and see if there are conflicts?

sure

Enubia commented:
@gnorbsl ready for review

gnorbsl commented on client/views/wishCards.ejs:
```diff
@@ -167,15 +173,14 @@
+                        if
+                        (locals.user)
+                        {
+                        %
```
your autoformat really fucks stuff up in the ejs files

gnorbsl replied:
aside fromt that this reverts whats in master right now
